assembly/src/parsers/u32_ops.rs

```diff
@@ -882,60 +921,67 @@ fn handle_u32_and_unsafe_check(
     Ok(())
 }
 
-/// Handles arithmetic operation that needs support for unsafe, full, operation and operation.n
+/// Handles arithmetic operation of checked, wrapping and overflowing types and its operation.n
 /// modes.
 ///
 /// Specifically handles these specific inputs per the spec.
-/// - Zero argument: assert the top two elements are u32 and push the result after  to the stack.
-/// - Single argument:
-///   - "unsafe" skips the assert check and directly performs the operation
-///   - "full" checks both numbers are u32 and perform the same operations as "unsafe"
-///   - Any number argument gets pushed to the stack, checked if both are u32 and performs the
-///     operation.
+/// - "Checked":
+///   - Zero argument: assert the top two elements are u32 and push the result after to the stack.
+///   - Single argument: any number argument gets pushed to the stack, checked if both are u32 and
+///     performs the operation.
+/// - "Wrapping":
+///   - Zero argument: skips the assert check and directly performs the operation, dropping high
+///     bits
+///   - Single argument: any number argument gets pushed to the stack, skips the assert check and
+///     directly performs the operation, dropping high bits
+/// - "Overflowing":
+///   - Zero argument: skips the assert check and directly performs the operation
+///   - Single argument: any number argument gets pushed to the stack, skips the assert check and
+///     directly performs the operation
```

Comment:
Let's update the text as follows:

- Checked: fails if either of the inputs or the output is not a u32 value.
- Wrapping: does not check if the inputs are u32 values (the immediate value is also not checked); overflow or underflow bits are discarded.
- Overflowing: does not check if the inputs are u32 values (the immediate value is also not checked); overflow or underflow bits are pushed onto the stack.